src/Components/Navbar.js

```diff
@@ -1,25 +1,61 @@
 import './Navbar.css'
 
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
 import PropTypes from 'prop-types'
 
+import { Link } from 'react-router-dom'
+
 import { Menubar } from 'primereact/menubar'
 
 function Navbar({ items, start, end }) {
+  const [version, setVersion] = useState('')
+
+  useEffect(() => {
+    fetch('/app.json')
+      .then((response) => response.json())
+
+      .then((data) => setVersion(data.version))
+
+      .catch((error) => {
+        console.log('Navbar useEffect', error)
+
+        alert('An error occurred please try again later.')
+      })
+  }, [])
+
   if (!end) {
     end = (
-      <a href="https://github.com/EddieHubCommunity/LinkFree" aria-label="View source on GitHub">
-        <i className="pi pi-github" style={{ fontSize: '2em' }}></i>
-      </a>
+      <div className="p-d-flex p-jc-center p-ai-center p-pr-2">
+        <Link
+          to={{ pathname: 'https://github.com/EddieHubCommunity/LinkFree' }}
+          target="_blank"
+          className="p-mr-2"
+          aria-label="LinkFree repository on GitHub"
+        >
+          <i className="pi pi-github" aria-hidden="true"></i>
+        </Link>
+
+        <div>v{version}</div>
+      </div>
     )
   }
 
-  return <Menubar model={items} start={start} end={end} className="p-mb-4" />
+  return (
+    <Menubar
+      model={items}
+      start={start}
+      end={end}
+      className="p-mb-4 p-flex-wrap p-jc-center"
+    />
+  )
 }
 
 Navbar.propTypes = {
   items: PropTypes.array,
+
```

Comment:
??

Reply:
@krishguptadev can you leave a more specific comment that helps @FidalMathew know what to fix.

Reply:
@emmalearnscode I was curious why these lines were added. 😅

Reply:
@krishguptadev Then ask why the lines have been added, it makes it easier to understand and know what to fix :)